### PR TITLE
fix: Show only NVIDIA drivers from Pop

### PR DESCRIPTION
**Pull request by mmstick**

### CODE REVIEW

File: src/Core/UbuntuDriversBackend.vala [hunk 1]
```diff
@@ -85,9 +84,13 @@ public class AppCenterCore.UbuntuDriversBackend : Backend, Object {
...
             }
 
-            // Filter out the nvidia server drivers
-            if (package_name.contains ("nvidia") && package_name.contains ("-server")) {
-                continue;
+            // Filter NVIDIA packages from Ubuntu
+            if (package_name.contains ("nvidia")) {
+                var name = package_name.split (",")[0];
+                if (!yield packaged_by_pop(null, name)) {
+                    stderr.printf("%s not packaged by Pop\n", name);
```
Comment (13r0ck) [inline src/Core/UbuntuDriversBackend.vala:91]: Wont this make pop shop print an error every time it checks for packages?

Comment (mmstick) [reply src/Core/UbuntuDriversBackend.vala:91]: If attached to a terminal, it'd print on stderr

**Review by 13r0ck — APPROVED** [review]

**Review by n3m0-22 — APPROVED** [review]
I'm no longer seeing the NVIDIA drivers from Ubuntu. All testing passed.

[shop-testing.txt](https://github.com/pop-os/shop/files/9855967/shop-testing.txt)

**Comment by jackpot51** [comment]
Please do a repo-release PR as soon as this builds in master staging.

**Pull request merged**